datatypes/src/collections/feature_collection.rs

```diff
@@ -1646,10 +1646,10 @@ mod tests {
             empty_hash_map_size
         );
 
-        let struct_stack_size = 144;
+        let struct_stack_size = 168;
```

Comment:
why did this change?

Reply:
Maybe they added a field to the struct or changed some internals :shrug: .